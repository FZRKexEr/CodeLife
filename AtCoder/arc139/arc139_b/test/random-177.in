100
30 9 14 10 8 17
70 11 16 13 8 14
35 12 11 20 18 3
26 3 3 9 12 9
65 8 15 5 9 7
24 16 10 18 10 10
84 1 1 1 18 1
3 12 10 6 10 14
8 6 9 4 2 5
9 9 13 2 7 17
65 5 12 14 19 1
65 14 7 1 13 20
65 13 1 13 17 19
24 5 13 1 20 8
81 4 12 2 10 5
95 15 9 19 19 13
92 5 3 1 12 2
3 5 9 3 9 19
36 5 17 13 15 10
37 7 18 12 3 3
40 11 6 8 1 16
5 19 4 17 10 9
71 2 6 15 18 9
51 9 20 3 17 3
57 3 2 9 9 3
4 8 12 4 20 13
46 2 4 5 17 4
53 14 16 2 5 4
48 2 2 20 8 18
44 3 10 14 17 8
31 17 4 15 5 19
60 13 9 14 13 7
55 4 19 19 20 18
42 18 9 16 15 4
47 13 19 19 9 14
7 6 2 13 14 20
11 2 3 3 19 6
62 18 12 15 6 12
62 20 17 14 6 16
44 8 8 3 15 5
70 3 3 9 14 5
13 4 19 20 16 20
57 13 15 1 16 9
91 7 1 9 20 7
66 20 19 17 13 3
63 18 17 12 4 13
88 13 8 18 1 12
24 19 4 2 10 1
67 10 8 15 5 9
71 18 19 18 14 7
24 4 4 12 17 9
80 8 16 16 17 13
49 2 3 12 3 17
1 8 10 1 12 7
27 3 3 18 9 1
39 9 12 16 10 9
67 13 3 2 11 8
70 13 11 17 20 1
63 13 6 19 20 18
63 8 17 3 17 20
63 7 5 12 8 2
93 9 14 3 20 9
65 12 12 2 13 1
35 15 9 13 16 1
69 2 3 17 13 10
27 10 18 9 7 11
47 16 10 4 7 1
23 2 12 15 17 12
58 11 10 9 4 2
50 6 4 19 4 19
58 2 18 13 18 9
78 14 2 8 17 10
35 20 8 7 7 16
29 6 3 17 5 11
75 19 3 15 5 17
43 9 2 4 4 17
17 16 1 11 14 11
47 10 18 9 4 5
2 10 3 2 13 13
93 19 20 6 3 7
51 9 17 5 16 6
74 13 18 7 1 14
94 2 3 8 20 18
35 9 17 4 15 6
72 19 2 17 7 19
57 5 4 3 10 3
27 17 18 12 16 7
55 5 4 11 2 16
22 13 4 20 18 16
81 13 12 18 19 1
79 3 17 19 13 11
19 5 20 5 6 17
70 11 15 18 1 14
13 14 5 9 3 9
95 20 6 12 3 10
29 5 5 8 2 6
42 4 5 2 7 6
21 13 13 4 18 8
38 2 13 19 12 4
5 7 19 8 9 16
