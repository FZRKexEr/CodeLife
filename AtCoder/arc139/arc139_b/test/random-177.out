54
58
39
90
53
92
84
18
10
7
24
61
85
19
48
159
62
9
46
18
27
26
150
29
90
16
54
22
192
63
92
76
135
112
70
27
24
66
46
37
124
68
39
260
141
120
54
6
58
273
54
65
84
1
9
39
59
44
132
89
42
74
15
44
230
77
14
112
47
66
63
125
72
65
117
44
25
23
4
106
18
46
566
16
171
45
115
22
70
168
121
38
19
45
68
42
52
40
80
40
